jina/types/arrays/buffer.py

```diff
@@ -0,0 +1,97 @@
+from collections import OrderedDict
+from typing import Union
+
+from ... import Document
+
+if False:
+    from .memmap import DocumentArrayMemmap
+
+
+class BufferPoolManager:
```

Comment:
given that `buffer` means binary string in Python context, i would rename this module as `buffer_pool.py` or `bpm.py`

Reply:
renamed